features/admin.users.v1/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -210,6 +206,15 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
     const { getLink } = useDocumentation();
     const config: ValidationFormInterface = getUsernameConfiguration(validationData);
     const isAlphanumericUsername: boolean = config?.enableValidator === "true";
+    const fileImportTimeout: number = useSelector((state: AppState) =>
+        state.config.ui.features.bulkUserImport.fileImportTimeout);
+    const userLimit: number = useSelector((state: AppState) =>
+        state.config.ui.features.bulkUserImport.userLimit);
+    const csvFileProcessingStrategy: CSVFileStrategy = new CSVFileStrategy(
```

Comment:
let's get the value for `csvFileProcessingStrategy` using useMemo hook, since it depends on the global `userLimit` state variable.

eg: 

```js
const csvFileProcessingStrategy = useMemo( () => {
	return new CSVFileStrategy(args_go_here)
}, [ userLimit ])
```

Reply:
Updated with [cfb98d1](https://github.com/wso2/identity-apps/pull/5939/commits/cfb98d12e9c1a1016a01ebb9159e28612a53880a)